docs/sources/panels-visualizations/visualizations/geomap/index.md

```diff
@@ -282,6 +282,16 @@ The GeoJSON layer allows you to select and load a static GeoJSON file from the f
 - **Add style rule** creates additional style rules.
 - **Display tooltip** allows you to toggle tooltips for the layer.
 
+{{% admonition type="note" %}}
+Styles can be set within the "properties" object of the GeoJSON with support for the following geometrys.
+
+- Polygon, MultiPolygon
+  - **"fill"** the color of the interior of the polygon
+  - **"fill-opacity"** the opacity of the interior of the polygon
+  - **"stroke-width"** the width of the line component of the polygon
```

Comment:
Do we need both the quotation marks and the bold? If these are properties in the JSON object, I would suggest the following formatting: 
```suggestion
  - `fill` - The color of the interior of the polygon
  - `fill-opacity` - The opacity of the interior of the polygon
  - `stroke-width` - The width of the line component of the polygon
```

Reply:
Adding the hyphen and capitalization, but keeping the double quotes to be consistent with the geojson standard.